mcode/pims
# Fix crash with rendering 'auth_number'

## Describe your changes

> Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

In the PIMS frontend, clicking the "View ETASU" button crashes the page on dev (after version 1.0.0 of pims) if an ETASU element that refers to the `auth_number` is attempted to be rendered. This PR filters out ETASU elements whose `name === 'auth_number'`, so the checkboxes/x's can render in the modal. 

**Screenshot of crash:**
<img width="1840" alt="image" src="https://github.com/mcode/pims/assets/33106214/00dc1211-bb2f-4d2c-8661-a6065290399d">


## Issue ticket number and Jira link

> Please include the Jira Ticket Number and Link for this issue/task.

N/A, this was randomly discovered. 

## Checklist before requesting a review

## Checklist for conducting a review

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

## How to manually test this PR
1. Select Jon Snow & Turalio in request-generator
2. Send the Rx to PIMS
3. Go to PIMS frontend at http://localhost:5050/DoctorOrders
4. Open the Network tab in DevTools
5. Click the View ETASU button. You will notice the modal renders without a crash, and in the metRequirements network request, the metRequirements list should still contain the object with the auth_number.
6. Proceed with the rest of the Turalio workflow. The modal should still render with the filled out ETASU.

![image](https://github.com/mcode/pims/assets/33106214/f1735c45-51b4-45e8-b780-248268f10fc0)
